migration-guides/11.0.md

```diff
@@ -59,3 +59,73 @@ The removal the timezone is now an 'opt-in' mechanism triggered by the `x-local-
 
 > [!NOTE]
 > You can revert to the previous behavior thanks to the global property `useLegacyDateExtension`
+<!-- generated deprecated - start -->
```

Comment:
generated with command `yarn eport-deprecates 11`

Reply:
is there a documentation where we can add this script usage example?

Reply:
Added a readme file